content/en/developers/integrations/create_a_tile.md

```diff
@@ -191,6 +191,8 @@ Once you have created a `README.md` file, add the following sections as H2s (`##
 | Data Collected  | Specify the types of data collected by your integration (if applicable), including metrics, events, service checks, and logs. Metrics added to the `metadata.csv` file automatically appear in this tab. <br><br> If your offering does not provide any of this data, you do not need to add a Data Collected section. |
 | Support | Provide contact information that includes an email to your Support team, a link to your company's documentation or blog post, and additional help information in a bulleted list format. |
 
+When adding links to the `README.md` file, format them using [footnote-style references][30]. For example, instead of embedding the URL directly in the text, write `see the [official Datadog documentation][1]` and define the link reference at the bottom of the file like `[1]: https://docs.datadoghq.com/`.
```

Comment:
```suggestion
When adding links to the `README.md` file, format them using [reference-style links][30]. For example, instead of embedding the URL directly in the text, write `see the [official Datadog documentation][1]` and define the link reference at the bottom of the file like `[1]: https://docs.datadoghq.com/`.
```
```suggestion
When adding links to the `README.md` file, format them using [footnote-style references][30]. For example, instead of embedding the URL directly in the text, write `see the [official Datadog documentation][1]` and define the link reference at the bottom of the file like `[1]: https://docs.datadoghq.com/`.
```